### ISSUE TITLE
[NVIDIA GeForce Now Library] Some Owned Games not Registering in Playnite 

### ISSUE THREAD

**Issue by Madmick2963** [darklinkpower/PlayniteExtensionsCollection#245]
### Extension name

NVIDIA GeForce Now Library

### Bug Description

I suspect I'm not even reporting this correctly or constructively, but I just double-checked all my Epic games in my Playnite Library to see which were now on the GeForce Now service, and I noticed five weren't registering in Playnite as owned/launchable even though I can play them from the GeForce Now app directly: 

1. Absolute Drift
2. FTL: Faster Than Light (Advanced Edition)
3. Neon Abyss
4. Shadowrun: Hong Kong
5. Shenmue III

I tried removing them from my library in GeForce Now, relaunching Playnite, closing Playnite, then re-adding them in GeForce Now, and launching Playnite again, but that didn't fix it. 

### To Reproduce

Playnite is failing to link them every time. 

### Extensions log

I looked in the ~/AppData/Local/Playnite folder but there was no Extensions.log file anywhere in that folder, or its subdirectories. The extension itself in Playnite directs me to a "Data" folder which is in the ~/Appdata/Roaming/Playnite/ExtensionsData folder, but there are only .json files there. 

Closest thing I can find is a .txt file named "Extensions" in the ~/AppData/Roaming folder. Is that what you need?

**Comment by darklinkpower** [comment]
The issue is because for Epic Library games, detection is done using the game name. To fix it you can open the database browser from MainMenu -> Extensions -> NVIDIA Geforce Now -> Open database browser. Then find the name of your game, right click, copy the correct name and change the name of the game in Playnite to that one

![image](https://user-images.githubusercontent.com/1389286/167510323-de8f523d-d8d2-417e-a99b-710fcbca38f8.png)

Afterwards it should be detected without issues

> Closest thing I can find is a .txt file named "Extensions" in the ~/AppData/Roaming folder. Is that what you need?

As mentioned in the issue template description, if you copy-paste `%AppData%\Playnite` it will take you to the correct directory

**Comment by cpobdev** [comment]
This doesn't seem to work for `Assassin's Creed® II` for Uplay.

The Database browser only lists Steam as an option
![image](https://user-images.githubusercontent.com/9098387/169720480-376ffe95-3770-49cd-b77a-fc0e3352f1d9.png)

When Geforce Now definitely supports both
![image](https://user-images.githubusercontent.com/9098387/169720503-47b55984-7cf1-4479-bb86-1ddab354db3c.png)

But I guess we're now at the mercy of what the `https://public.games.geforce.com/graphql` endpoint is returning?

**Comment by darklinkpower** [comment]
> This doesn't seem to work for `Assassin's Creed® II` for Uplay.
> 
> The Database browser only lists Steam as an option ![image](https://user-images.githubusercontent.com/9098387/169720480-376ffe95-3770-49cd-b77a-fc0e3352f1d9.png)
> 
> When Geforce Now definitely supports both ![image](https://user-images.githubusercontent.com/9098387/169720503-47b55984-7cf1-4479-bb86-1ddab354db3c.png)
> 
> But I guess we're now at the mercy of what the `https://public.games.geforce.com/graphql` endpoint is returning?

Yes. The endpoint used by the extension is https://public.games.geforce.com/graphql?query={apps(vpcId:"NP-SEA-01",language:"en_US",first:1300,after:"MA=="){numberReturned,pageInfo{hasNextPage,endCursor},items{id,cmsId,title,type,variants{id,title,appStore,gfn{status,releaseDate}osType,storeId}}}}

I tried investigating if it was possible to get the missing items but I could not find out if it was possible.
